server/events/post_workflow_hooks_command_runner.go

```diff
@@ -89,13 +102,33 @@ func (w *DefaultPostWorkflowHooksCommandRunner) runHooks(
 	repoDir string,
 ) error {
 
-	for _, hook := range postWorkflowHooks {
-		_, err := w.PostWorkflowHookRunner.Run(ctx, hook.RunCommand, repoDir)
+	for i, hook := range postWorkflowHooks {
```

Comment:
After reviewing this code, it seems like the post and pre workflow hooks are very similar. Perhaps in the future we can combine them so we do not need to copy pasta between post and pre golang files. This is probably what you were getting at with your follow up refactor pr.

Reply:
Yup, you got it right, I would also like to decouple the code from the `jobs` package, I had to couple it together to add the details link functionality.

Reply:
Ok let's refractor in a follow up pr